server/rooms.ts

```diff
@@ -1058,6 +1058,7 @@ export class BasicChatRoom extends BasicRoom {
 	readonly creationTime: number | null;
 	readonly type: 'chat' | 'battle';
 	minorActivity: Poll | Announcement | null;
+	queuedActivity: Poll[] | null;
```

Comment:
AFAICT, this is never null.
```suggestion
	queuedActivity: Poll[];
```

(Don't forget to remove all the unnecessary non-null assertion and optional chaining.)